### ISSUE TITLE
There is no sigmoid function

### ISSUE THREAD

**Issue by xuxiaoyuxuxiaoyuxuxiaoyu** [xxradon/PytorchToCaffe#18]
I tried to convert my pytorch model to caffe but failed in a block which contains a sigmoid function

**Comment by xxradon** [comment]
Yes,there are so many layers that I do not add to this repo.Maybe you can do it by yourself.

**Comment by xiezheng-cs** [comment]
@xuxiaoyuxuxiaoyuxuxiaoyu 
How to deal with the problem when a block contains a sigmoid function ???
